script/getVersions.js

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env node
+
+const semver = require('semver')
+const superagent = require('superagent')
+
+const url = 'https://api.github.com/repos/nodejs/node/releases'
```

Comment:
This is a better source, and won't hit GitHub rate limits https://nodejs.org/dist/index.json

Reply:
Great thing, I didn't know about it, but looks good.